source/css/sass/_normalize.scss

```diff
@@ -64,6 +64,9 @@ ul,
 ol {
   @include blg-spacing("block-end", "small", "margin");
   margin-top: 0;
+  padding-left: 0;
+  // Overflow auto is so the bullet will be still read from a screen reader and will push the bullet off screen
+  overflow: auto;
```

Comment:
IIRC we weren't going to remove bullets on lists by default. We should create a mixin for doing so.

Reply:
I have created a mixin so bullets or numbers etc can be removed